uber/models/mivs.py

```diff
@@ -214,6 +228,20 @@ class IndieDeveloper(MagModel):
     email = Column(UnicodeText)
     cellphone = Column(UnicodeText)
 
+    @property
+    def email_to_address(self):
+        # Note: this doesn't actually do what we want right now
+        # because the IndieDeveloper and attendee are not properly linked
+        if self.matching_attendee:
+            return self.matching_attendee.email
+        return self.email
+
+    @property
+    def cellphone_num(self):
```

Comment:
Since we don't use the cellphone in the attendee matching lookup, this actually totally does what we want, which is to return the matching attendee's cellphone number instead of the (uneditable post-acceptance) one on the IndieDeveloper object.